cli/Cargo.toml

```diff
@@ -68,6 +68,7 @@ birdcage = { version = "0.2.0" }
 libc = "0.2.135"
 ignore = { version = "0.4.20", optional = true }
 uuid = "1.4.1"
+dashmap = "5.5.0"
```

Comment:
I was pondering if the `dashmap` dependency was really worth it or if a simple locking `HashMap` would be better (there should be zero contention anyway). But I noticed that we depend on it already anyway, so I feel like it shouldn't be an issue.